neuropacks/peanut/peanut.py

```diff
@@ -140,7 +143,7 @@ def load_peanut(fpath, epoch, spike_threshold, bin_width=100, bin_type='time', b
     return dat
 
 
-class Peanut_SingleEpoch():
```

Comment:
@akumar01 Also have you been using this class at all, or is it safe to replace like this?